inc/alert.class.php

```diff
@@ -52,7 +52,9 @@ class Alert extends CommonDBTM {
    const ACTION      = 5;
    const PERIODICITY = 6;
 
-
+   /**
+    * @see CommonDBTM::prepareInputForAdd($input)
```

Comment:
I'm sorry but this is still wrong...

From the doc point of view, `@see` should be used to reference any other method/function; but not overridden one; since the generator will automatically add a link on overridden method. Also, it does not seem to generate a link with `@see` (I'd expect...)

With `@see`:
![class-alert see](https://cloud.githubusercontent.com/assets/224733/22140267/a6306d90-deeb-11e6-98e0-4c0926371b9d.png)


Without any docblock:
![class-alert nodocblock](https://cloud.githubusercontent.com/assets/224733/22140269/aa1a0682-deeb-11e6-86e1-26310a5ec74e.png)

And from a CS point of view, this is not valid :)

On my side, to validate the docblocks; I've cloned the standards project sources; and sdet my editor to use its master branch (since we cannot do it on glpi side right now).
To test output, get apigen installed and then run:
```
$ apigen generate -d apidoc -s inc/alert.class.php,inc/commondbtm.class.php
```

This will generate documentation for Alert class (you have to add commondbtm so the generator can find parent methods) in the apidocs directory.